src/components/pages/main-page/MainPage.tsx

```diff
@@ -38,114 +39,115 @@ const MainPage: FC<MainPageProps> = ({ data }) => {
+                        size={ButtonSize.LARGE}
+                      />
+                    </Link>
+                    <Divider sx={stylesMUI.buttonDivider} />
+                  </>
+                )}
+                <Link href={'/priority'}>
+                  <Button
+                    sx={stylesMUI.buttons}
+                    text={'Обрати пріоритет'}
+                    disabled={false}
+                    variant={ButtonVariant.OUTLINE}
+                    size={ButtonSize.LARGE}
+                  />
+                </Link>
+              </Box>
+              <Box sx={stylesMUI.buttonTab}>
+                {!isLoggedIn && (
+                  <>
+                    <Link href={'/contract'}>
+                      <Button
+                        sx={stylesMUI.buttons}
+                        text="Договір про навчання"
+                        disabled={false}
+                        color={ButtonColor.PRIMARY}
+                        variant={ButtonVariant.FILLED}
+                        size={ButtonSize.MEDIUM}
+                      />
+                    </Link>
+                  </>
+                )}
+                <Link href={'/priority'}>
+                  <Button
+                    sx={stylesMUI.buttons}
+                    text={'Обрати пріоритет'}
+                    disabled={false}
+                    variant={ButtonVariant.OUTLINE}
+                    size={ButtonSize.MEDIUM}
+                  />
+                </Link>
+              </Box>
+              <Box sx={stylesMUI.buttonMob}>
+                {!isLoggedIn && (
+                  <>
+                    <Link href={'/contract'}>
+                      <Button
+                        sx={stylesMUI.buttons}
+                        text="Договір про навчання"
+                        disabled={false}
+                        color={ButtonColor.PRIMARY}
+                        variant={ButtonVariant.FILLED}
+                        size={ButtonSize.SMALL}
+                      />
+                    </Link>
+                  </>
+                )}
+                <Link href={'/priority'}>
+                  <Button
+                    sx={stylesMUI.buttons}
+                    text={'Обрати пріоритет'}
+                    disabled={false}
+                    variant={ButtonVariant.OUTLINE}
+                    size={ButtonSize.SMALL}
+                  />
+                </Link>
+              </Box>
+            </Box>
+          </Box>
+          <Box sx={stylesMUI.buildImage}>
             <BannerImage />
-          </div>
-        </div>
-        <div className={styles['resources']}>
-          <h3>Студентські ресурси</h3>
-          <div className={styles['resource-card-container']}>
-            <div className={styles['resources-cards']}>
+          </Box>
+        </Box>
+        <Box sx={stylesMUI.resources}>
+          <Typography variant={'h3Bold'} sx={stylesMUI.resourcesH3}>
```

Comment:
if your font is changing by device with you don't need to set `variant={'h3Bold'}` as you have sx styles for it